src/overlays/actors/ovl_Boss_Va/z_boss_va.c

```diff
@@ -2820,7 +2820,7 @@ void BossVa_Update(Actor* thisx, PlayState* play2) {
 
             for (i = 2; i >= 0; i--) {
                 if ((play->envCtx.adjAmbientColor[i] - 1) > 0) {
-                    play->envCtx.adjAmbientColor[i] -= 1;
+                    play->envCtx.adjAmbientColor[i]--;
                 } else {
                     play->envCtx.adjAmbientColor[i] = 0;
                 }
```

Comment:
fwiw this matches as `play->envCtx.adjAmbientColor[i] = MAX(play->envCtx.adjAmbientColor[i] - 1, 0);`

I'll use this comment to discuss the `+/-=1 -> ++/--` changes in general too, I feel like it's justified in some spots to keep `+/-= 1`.
Typically where it denotes a discrete action, like taking away a key or non-trivially move an index by one, or things that just happen to change by 1 (but could change by more, like for animated values such as distances), as opposed to timers which increment/decrement continuously.
Then there's the weird spots like here where 1 appears in the conditional's `- 1` so I also wouldn't abstract it under a --

idk, I could go with the ++/-- everywhere but reluctantly

Reply:
```c
play->envCtx.adjAmbientColor[i] = MAX(play->envCtx.adjAmbientColor[i] - 1, 0);
```

Reply:
This doesn't seem to match for me?

Reply:
I tried again and
```c
play->envCtx.adjAmbientColor[i] = MAX(play->envCtx.adjAmbientColor[i] - 1, 0);
```
in place of
https://github.com/zeldaret/oot/blob/9f0b7bb8a3e84cedbe2eca82450113fe3b5b9b98/src/overlays/actors/ovl_Boss_Va/z_boss_va.c#L2823-L2827
does match for me :thinking: 
but, whatever for this PR

Reply:
Weird, yeah it works for me now too. I did it for all three blocks in this function